app/javascript/components/public_health/query/AdvancedFilter.js

```diff
@@ -652,43 +652,73 @@ class AdvancedFilter extends React.Component {
    */
   getRelativeTooltipString(filter, value) {
     const filterName = filter.title.replace(' (Relative Date)', '');
-    let statement, rangeString, start, end;
+    let rangeString, before, after;
+    let statement = '';
+    const operatorValue = value.operator.replace('-', ' ');
 
-    // set variables for date options including a time stamp
-    if (filter.hasTimestamp) {
+    if (value.operator === 'more-than') {
       if (value.when === 'past') {
-        rangeString = 'dated through today’s date';
-        start = moment()
+        before = moment()
           .subtract(value.number, value.unit)
+          .add(1, 'd')
           .format('MM/DD/YY');
-        end = 'now';
       } else {
-        rangeString = 'with today’s date as of the current time';
-        start = 'now';
-        end = moment()
+        after = moment()
           .add(value.number, value.unit)
+          .subtract(1, 'd')
           .format('MM/DD/YY');
       }
+    } else if (value.operator === 'less-than') {
+      // set variables for date options including a time stamp
+      if (filter.hasTimestamp) {
+        if (value.when === 'past') {
+          rangeString = 'dated through today’s date';
+          after = moment()
```

Comment:
This is specifically only for the "less than x in the past/future", the number seems to be off by one in the tooltip

![Screen Shot 2021-02-18 at 10 41 48 AM](https://user-images.githubusercontent.com/9956561/108381822-31f4d100-71d6-11eb-8386-f8343fe872e0.png)

Reply:
everything should match now